support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -200,7 +199,7 @@ function userInTest(
 }
 
 function randomNumber(mvtId: number, seed: number): number {
-	const rng = seedrandom(mvtId + seed);
+	const rng = seedrandom(`${mvtId + seed}`);
 	return Math.abs(rng.int32());
```

Comment:
Using string interpolation here as `@types/seedrandom` requires a string though `rng` is of type number

Reply:
Unexpectedly converting the argument passed to the `seedrandom` function to a string caused one of our tests to fail. We have looked into the library and though seedrandom accepts and converts an argument of type number, the eventual seed is slightly different than if it is passed a string. 

To account for this quirk from the library, the seed of the failing test which allocates a user to a `variant` group has been modified and to confirm functionality a test to allocate a user to the `control` group has been included.

Reply:
@marialani thanks for pointing me towards this. If we alter our use of `seedrandom` to pass a string then as you pointed out the eventual seed returened by that function will be slightly different than if it is passed a `number`. The issue with releasing this is that we'll potentially reassign users to different cohorts in any existing AB tests. 

I'd much rather use `"@types/seedrandom"` and pass a string as `seedrandom` intends and as you've done, but I'm not sure we can release this whilst we have any ab tests running?

Reply:
Yeah completely agree, I can remove the ab test related changes from this PR and maybe we can time making the change to the code between ab tests?

Reply:
OK I think that seems sensible, I'd say remove the `abtest.ts` changes from here and we can release them as a separate PR that doesn't risk compromising any live abtests.

Reply:
Alternatively could we ignore the TS errors related to `seedrandom` and have a `TODO` to fix these separately in a follow-up PR?